tqdm.py

```diff
@@ -0,0 +1,42 @@
+from time import sleep
+import datetime as dt
+import sys
+
+
+def Tqdm(iterable):
+    iteration_counter = 1
+    total_iterations = len(iterable)
+    start_time = dt.datetime.now()

```

Comment:
Use `time.monotonic` instead (new in Python 3.6)

Reply:
Done